### ISSUE TITLE
Accept commented code as correctly indented

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#14651]
## Steps to reproduce the problem

```ruby
123
  .then { 456 }
  .then { 789 }

# Comment the last line (`M-; / comment-dwin` in Emacs)

123
  .then { 456 }
  # .then { 789 }

```

## Expected behavior

Accept the commented code as correctly indented, this being the default way to comment out code.

## Actual behavior

`Incorrect indentation detected (column 2 instead of 0). [[Correctable] Layout/CommentIndentation]`

This is what it wants, which requires unwarranted adjustments to the code.

```ruby
123
  .then { 456 }
# .then { 789 }
```

## RuboCop version

```
System Version: macOS 26.0.1 (25A362)
Model Identifier: MacBookAir10,1
ruby      2025-08-01 30a20bc16
rubocop   2025-10-18 f7f653f50
```

**Comment by bbatsov** [comment]
Closing as stale — no engagement in 3+ months. Indentation of commented-out code is subjective and this didn't gain traction. Feel free to reopen if there's renewed interest.

**Comment by forthrin** [comment]
Sort of agree. The broader problem may actually be keeping Emacs Ruby modes in sync with a small handful Rubocop preferences, in particular indentation rules. Does anyone have a cheat sheet for this? Again, move to Discussions appreciated.

PS! Great you're catching up the back log (should be quick with < 200 issues). And see you found #14759 to assist you.
